manifest.json

```diff
@@ -24,8 +24,20 @@
             "suggested_key": {
                 "default": "Ctrl+Down"
             }
+        },
+        "toggle-new-tab-option": {
```

Comment:
Can you please change this to `toggle-add-background-tabs-after-current-option`? I know it's long and I don't think it's visible to the user in any way, but I think it would make naming more consistent.